catalyst/data/sampler.py

```diff
@@ -1,8 +1,13 @@
-from typing import Iterator, List  # isort:skip
+from operator import itemgetter
+from typing import Iterator, List
```

Comment:
Could you please return # isort:skip and move from typing import ... line to the top of the file?

Reply:
done